cmake/Modules/Packages/LATTE.cmake

```diff
@@ -1,4 +1,11 @@
 enable_language(Fortran)
+
+# using lammps in a super-build setting
+if(TARGET LATTE::latte)
```

Comment:
We could move this block into `FindLATTE.cmake` to hide it from the user, but I was planning to make use of https://github.com/lanl/LATTE/pull/112 in the future and drop our custom `FindLATTE.cmake`.

@rbberger any opinions?